### ISSUE TITLE
eliminate repo-root for good

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#24348]
Problem

Original problem is here #23987 

Solution

After #24193 which should be pending, delete the rest of repo-root dependent calls and simply bootstrap any needed repo artifacts using the `bindata_util.go` asset functionality i.e. `guestbookPath = filepath.Join(framework.TestContext.GeneratedAssetsDir, framework.GENERATED_ASSETS, "examples/guestbook")`.

**Comment by jayunit100** [comment]
cc @kubernetes/sig-testing @timothysc

**Comment by timothysc** [comment]
This would be a long-term goal, which I'm hoping will be viable for 1.3.

**Comment by spiffxp** [comment]
/close
Closing in favor of #31270
